tests/test_solution.py

```diff
@@ -576,3 +579,59 @@ def test_serialization(s1, s2, tmpdir):
     # also should point to different Store instances
     # TODO currently this test will fail due to a bug in maggma's __eq__
     # assert s2_new.database != s2.database
+
+
+def test_valid_preset():
+    solution = Solution.from_preset("seawater")
+    assert isinstance(solution, Solution)
+    assert solution.temperature is not None
+    assert solution.pressure is not None
+    assert solution.pH is not None
+    assert len(solution._solutes) > 0
```

Comment:
Can you make these tests more precise? I.e., make sure the pH has the correct quantitative value based on the .yaml contents

Reply:
Similar to the approach you took in `test_correct_data_loading`